llvm/lib/DebugInfo/LogicalView/Core/LVLocation.cpp

```diff
@@ -284,6 +284,7 @@ std::string LVOperation::getOperandsDWARFInfo() {
   case dwarf::DW_OP_implicit_value:
     Stream << "TODO: DW_OP_implicit_value";
     break;
+  case dwarf::DW_OP_GNU_implicit_pointer:
```

Comment:
Don't think the test added here exercises this codepath. Should this be handled in a separate LogicalView PR?